resources/telemetry.md

```diff
@@ -0,0 +1,521 @@
+and manually input variable values, instead of relying on Grafana/Tempo
+detecting them.
+
+![](/assets/images/telemetry/grafana_variable_definition.png)
+
+While configuring your variable, if you want to show multiple plots with
+different values of this variable, be sure to check the "Include All Option"
+checkbox.
+
+When finished defining your variable and its values, click the blue "Save
+dashboard" in the upper right. If you navigate away from this page without
+clicking this button, your variable will not be saved.
+
+### Queries
+
+[Queries provide the data from the data source to the panel/visualization](https://grafana.com/docs/tempo/latest/traceql/). Queries differ based on the data source being used. [Tempo's queries use TraceQL](https://grafana.com/docs/tempo/latest/traceql/).
+
+Attributes must be included as selectors in the query, or else they won't be available for filtering down the line. The TraceQL line can get pretty long:
+
+{% raw %}
+```
+{} | select(span:duration, name, resource.rapids.labels, resource.service.name, resource.rapids.cuda, resource.rapids.cuda, resource.git.job_url, resource.rapids.py, resource.rapids.gpu, resource.rapids.package_type, resource.rapids.arch, resource.rapids.linux, resource.rapids.driver, resource.rapids.deps,span.sccache.main_process.hit_rate,span.sccache.main_process.miss_rate,span.sccache.main_process.error_rate)
+```
+{% endraw %}
+
+![](/assets/images/telemetry/panel_query.png)
+
+On the right side, take note of the "Repeat options" - this is how you get multiple plots (one per variable value).
+
+On the bottom, take note of the limits and table format. For limits, each trace
+represents one top-level workflow run. Scale this accordingly, especially if you
+are looking back far in time. The span limit is less important - you just want
+this number high enough to capture the largest number of spans for a given
+trace. It's fine to set it to a high number, such as 1000 spans. Regarding table
+format, we are interested in spans, not traces. You will not see the additional
+metadata columns in the trace table format, because they apply to the spans.
+
+As always, remember to click the upper-right save button after making any
+changes.
+
+##### Filtering and grouping data
+
+We store our span data using Grafana Tempo. Tempo allows TraceQL queries for filtering data. In general, there are two parts of [a TraceQL query](https://grafana.com/docs/tempo/latest/traceql/#traceql):
+
+1. The `{filter}` part, which is used to filter the traces
+2. The `select` part, which is used to select the metadata fields to return
+
+The query is limited to a certain number of traces and spans per trace, as well as the time range of the query. The maximum time range is 7 days.
+
+The resource-level attributes that are captured can be found in shared-workflows, such as https://github.com/rapidsai/shared-workflows/blob/branch-25.04/.github/workflows/conda-cpp-build.yaml#L117C32-L117C199.
```

Comment:
Maybe update these permalinks to a newer branch.

Reply:
```suggestion
The resource-level attributes that are captured can be found in shared-workflows, such as https://github.com/rapidsai/shared-workflows/blob/d94b26cdfafe306cde800e8cc11ed9444a21c634/.github/workflows/conda-cpp-build.yaml#L158.
```